docs/usage/deprecated-variables.rst

```diff
@@ -1,127 +1,127 @@
 Deprecated variables
-^^^^^^^^^^^^^^^^^^^^
+--------------------
```

Comment:
This file is included. Please don't change the heading level.

Reply:
Damned... will need to put on my thinking hat then... as the heading actually was the issue for why I changed them. (just don't recall where i spotted it)

Reply:
Now I remember and I will encourage you to try to make a local build your self, to see the new index ordering under `Usage`

Personally I think it is looking way better AND it make it easier to find the "Deprecated variables" as you no longer need to go all the way through "From a terminal"

It is also easier to link to a specific old value under issues and / Or discussions

![image](https://user-images.githubusercontent.com/44526987/127860864-4ecff83c-3920-4620-b966-442d83734ce6.png)

![image](https://user-images.githubusercontent.com/44526987/127860991-aa2226e1-9f77-4720-b08f-e568b357fbb5.png)

![image](https://user-images.githubusercontent.com/44526987/127861146-9b0f6b84-05c5-43f5-8a66-d7efdf1d423b.png)

Try it your self and let me know what you think when you have tried it :smirk: 

I put a :ice_cream: on the line your will change your mind :smiley:

Reply:
The problem is that they are not variables ... They are CLI arguments @spirillen ... 

That's why they are in there.... 

We have 2 options ... 

1. Reverse as it was.
2. List the old and new configuration keys along with the `--arguments`.

Reply:
3. Rename `deprecated-variables.rst` to `deprecated-arguments.rst`
    1. Rename plus keeping all arguments collected in one file
        1. Pitfall is the long list of "useless" arguments vs
        2. One list of ACTIVE arguments
        3. One list of deprecated arguments

This would actually be better as you mention yourself, they are arguments, not variables.

Reply:
> The problem is that they are not variables ... They are CLI arguments @spirillen ...
> 
> That's why they are in there....
> 
> We have 2 options ...
> 
>     1. Reverse as it was.
> 
>     2. List the old and new configuration keys along with the `--arguments`.

Experimental filenaming for deprecated args
https://mypdns.org/spirillen/PyFunceble/-/commit/008368aee770e73aaca5c7da349337a20c51f538